src/app.ts

```diff
@@ -35,6 +36,23 @@ const shouldStartMinimized =
   app.commandLine.hasSwitch('launch-minimized') ||
   config.get(ConfigKey.LaunchMinimized)
 
+const trayIcon = nativeImage.createFromPath(
```

Comment:
I also just thought that creating the native image could be extracted to a helper function that takes a boolean of unread true/false as that is the only difference between the variables.

Reply:
Yeah, good point. I'll make the code change.